services/src/test/java/org/keycloak/protocol/TestAuthenticatedClientSessionModel.java

```diff
@@ -0,0 +1,119 @@
+package org.keycloak.protocol;
+
+import org.keycloak.models.AuthenticatedClientSessionModel;
+import org.keycloak.models.ClientModel;
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.UserSessionModel;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestAuthenticatedClientSessionModel implements AuthenticatedClientSessionModel {
```

Comment:
LGTM, thanks @tomjo.

@mposolda The only thing I'm wondering is whether we actually run this test anywhere. I checked our GHA and it doesn't seem so.